### ISSUE TITLE
Define manual testing practices

### ISSUE THREAD

**Issue by nkkl** [18F/cms-hitech-apd#68]
- [x] accessibility
 - [x] usability and experience
 - [x] product robustness

**Comment by nkkl** [comment]
Jerome and I are meeting about this later today. 👍

**Comment by nkkl** [comment]
Bug reporting process:
- (find a bug)
- File a new issue about the bug
  - Label it as a bug
  - Don't add to backlog; bugs will get added to the task board during triage
  - Write steps to reproduce in the body of the bug
  - Mark the severity of the bug (sev 1: experience *cannot* be completed, sev 2: workaround exists, sev 3: experience is unpolished)

Triage process:
- Mark the priority of fixing the bug
- Pull into backlog as appropriate and assign to someone to fix

**Comment by nkkl** [comment]
Manual test process:
- Create a checklist of scenarios based on our use cases
- Run through checklist once a sprint
- Play with features as they're released
- As bugs are found, file them according to our bug filing process

**Comment by nkkl** [comment]
Full WCAG 2.0 based review once per milestone.
